### PR TITLE
Update incuna_test_utils

### PR DESCRIPTION
**Pull request by adam-incuna**
There's some nice new file-related stuff in incuna-test-utils that we can use in here to clean the code up somewhat.

@incuna/backend quick review please?

### CODE REVIEW

File: groups/tests/test_forms.py [hunk 1]
```diff
@@ -36,10 +37,11 @@ def test_form_fields(self):
         self.assertCountEqual(fields, expected)
 
     def test_form_valid(self):
-        file = factories.FileCommentFactory.create().file.file
-        data = {'file': file}
+        f = uploadable_file()
+        f.name = 'image.png'
```
Comment (meshy) [inline groups/tests/test_forms.py]: Should this be defined in incuna-test-utils?

Comment (adam-incuna) [inline groups/tests/test_forms.py]: Oh, whoops.  It already is, but I made this change while experimenting on the way to doing it in `incuna-test-utils`. Thanks!

Comment (meshy) [inline groups/tests/test_forms.py]: :+1:

Comment (adam-incuna) [inline groups/tests/test_forms.py]: Updated :)

**Comment by meshy** [comment]
:+1:

**Pull request merged**